src/app/phast/losses/operations/co2-savings-phast/co2-savings-phast.component.ts

```diff
@@ -213,7 +215,7 @@ export class Co2SavingsPhastComponent implements OnInit {
     if (isUserFormChange || !this.form.controls.userEnteredBaselineEmissions.value) {
       let selectedFuelOutputRate: number = _.find(this.fuelOptions, (val) => { return this.form.controls.fuelType.value === val.fuelType; }).outputRate;
```

Comment:
Might as well make a getter method for 216-223 and the comparable lines in the other method